frontend/lib/src/components/elements/CodeBlock/CopyButton.tsx

```diff
@@ -51,7 +51,7 @@ class CopyButton extends PureComponent<Props> {
   public render(): ReactNode {
     return (
       <StyledCopyButton
-        data-testid="stCopyButton"
+        data-testid="stCodeCopyButton"
```

Comment:
sanity-check: should this have a `className` attribute?

Reply:
Yep, for this project we only want to guarantee a stable class name on the top-level div of elements.